### ISSUE TITLE
Browsers can't open IPv6 addrs

### ISSUE THREAD

**Issue by ahmetalpbalkan** [yudai/gotty#42]
```
➜  ~  gotty -w top
2015/08/23 12:20:01 Server is starting with command: top
2015/08/23 12:20:01 URL: http://::1:8080/
2015/08/23 12:20:01 URL: http://127.0.0.1:8080/
2015/08/23 12:20:01 URL: http://fe80::1:8080/
2015/08/23 12:20:01 URL: http://fe80::a65e:60ff:fecc:5cc7:8080/
2015/08/23 12:20:01 URL: http://192.168.0.113:8080/
2015/08/23 12:20:01 URL: http://fe80::6048:20ff:fea9:ec6f:8080/
2015/08/23 12:20:01 URL: http://192.168.99.1:8080/
```

From what observe on Chrome and Safari on OS X is browsers can't open addrs like  `http://::1:8080/` or `http://fe80::a65e:60ff:fecc:5cc7:8080/` at all. Maybe the program should just print IPv4 ones?

**Comment by yudai** [comment]
Thank you for the report.
I realized browsers cannot open IPv6 addresses without the delimiters (`[` and `]`).
Now it shows URLs with the delimiters and you can open them on your browser.

**Comment by ahmetalpbalkan** [comment]
@yudai left a comment at https://github.com/yudai/gotty/commit/5d2cdd2768a091928892d99a106a7791df1ee244

also you should probably mention issue numbers (#nn) in the commit messages so that it would be easy to track/associate.

**Comment by yudai** [comment]
@ahmetalpbalkan Thanks! Replaced the logic with `net.JoinHostPort `.
Yeah, I often forget to add #nn in commits :)

**Comment by ahmetalpbalkan** [comment]
@yudai yeah you can even close issues automatically with commit messages such as `Fixes #42`. :)

**Comment by NatoBoram** [comment]
Looks like this issue was closed 4 years ago, but I just reinstalled `gotty` and I still have this issue.

```log
2019/02/17 17:00:09 Alternative URL: http://fe80::6631:50ff:fe5b:b589:5916/
```
